### DISCUSSION TITLE
Two Javamelody collector servers.

### DISCUSSION THREAD

**Discussion by SiddGhoshGit** [javamelody/javamelody#1111] [General]
I have a question , For the sake of High availability / redundancy . Can two javamelody collector servers collect data from one remote application server without any conflicts ?

I mean for example we have two collector servers A and B and I add remote server C to be monitored in the app.properties file in both collector Servers A and B , therefore we will have two redundant sets of data correct ? and this should not cause any conflict ?

Please advise.

**Comment by evernat** [comment]
Sorry for the delay.
No, you can't do that, as said the [advanced user's guide](https://github.com/javamelody/javamelody/wiki/UserGuideAdvanced#optional-centralization-server-setup) : 
> An application must not be monitored from two collector servers.
